server.js

```diff
@@ -16,20 +16,21 @@
 
 'use strict';
 
-const fs = require('fs')
-const https = require('https')
-const fetch = require('node-fetch')
-const { Strategy, Issuer, custom } = require('openid-client')
-const passport = require('passport')
-const express = require('express')
-const session = require('express-session')
-const config = require('./config')
-
-const env = process.env.ENVIRONMENT
-console.log(`Environment: ${env}`)
-
+const fs = require('fs');
+const https = require('https');
+const fetch = require('node-fetch');
+const { Strategy, Issuer, custom } = require('openid-client');
+const passport = require('passport');
+const express = require('express');
+const session = require('express-session');
+const config = require('./config');
+
+const env = process.env.ENVIRONMENT;
+console.log(`Environment: ${env}`);
+
+(async () => {
 // Configure the OpenID Connect client based on the issuer.
-const issuer = new Issuer(config.issuer[`garden-${env}`]);
+const issuer = await Issuer.discover(config.issuer[`garden-${env}`]);
```

Comment:
**praise:** I was able to run this and get the issuer that I expected (_https://digital.garden-fi.com/a/consumer/api/v0/oidc_) for this FI and the claims (including _https://api.banno.com/consumer/claim/institution_id_) that I expected for my user. 👍